api/jobset/v1alpha1/jobset_types.go

```diff
@@ -89,6 +90,7 @@ type ReplicatedJobStatus struct {
 }
 
 // +genclient
+// +k8s:openapi-gen=true
```

Comment:
@danielvegamyhre IIUC, we must add this marker to doc.go instead of adding this marker to types.

https://github.com/kubernetes/code-generator/blob/e60f5637f0bf9de7253d9258b4cb0b8958e2c5a1/examples/apiserver/apis/example/v1/doc.go#L17

Reply:
that worked, thanks!

Reply:
👍